server.js

```diff
@@ -578,4 +579,113 @@ app.get('/graph', function (req, res) {
       });
   });
 
+/**
+ * Get account info
+ */
+app.get('/account/:account', function (req, res){
+    let req_account = req.params.account;
+    axios.all([
+        axios.get(SGAPI + '/account/' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/account/' + req_account + '/rewards', { httpsAgent: agent }),
+        axios.get(SGAPI + '/delegations?delegatorAddress=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=20&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=40&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=60&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(SGAPI + '/transactions?offset=80&sender=' + req_account, { httpsAgent: agent }),
+        axios.get(CGAPI + '/simple/price?ids=cosmos&vs_currencies=usd', { httpsAgent: agent }),
+    ])
+        .then(axios.spread(function (data1, data2, data3, data4, data5, data6, data7, data8, data9) {

```

Comment:
suggestion, rename these variables to have some context. FOr instance:
- data1 could be availabilityData
- data2 could be delegatedData
- etc.

This makes it a little more readable by giving the variables a little more context.